studio/schemas/documents/companyLocations.ts

```diff
@@ -1,16 +1,16 @@
 import { defineField } from "sanity";
 
-export const companyLocationsID = "companyLocations";
 export const companyLocationID = "companyLocation";
+export const companyLocationNameID = "companyLocationName";
 
 const companyLocations = defineField({
```

Comment:
If we change name to companyLocation, should not classname and function change as well?

Reply:
Yep, thought I did that, will fix :+1: